carbon/assetlist.json

```diff
@@ -3,28 +3,228 @@
     "chain_name": "carbon",
     "assets": [
       {
-        "description": "The native token of Carbon",
+        "description": "The native governance token of Carbon",
         "denom_units": [
           {
             "denom": "swth",
-            "exponent": 0,
-            "aliases": []
-          },
-          {
-            "denom": "dswth",
             "exponent": 8,
```

Comment:
we require a denom_unit with 0 exponent, and it should be the true minimal denom, 'swth', and not anything else.